lib/proteus/kit.rb

```diff
@@ -16,7 +16,15 @@ def new(kit_name, repo_name = nil)
 
       if system "git ls-remote #{url(kit_name)} #{repo_name} > /dev/null 2>&1"
         puts "Starting a new proteus-#{kit_name} project in #{repo_name}"
-        system "git clone #{url(kit_name)}#{' ' + repo_name}"
+        system %{
+          git clone #{url(kit_name)} #{repo_name} &&
```

Comment:
Can you quote the arguments?

* `git clone "#{url(kit_name)}" "#{repo_name}"`
* `cd "#{repo_name}"`